.sync.yml

```diff
@@ -0,0 +1,12 @@
+:global:
+  componentName: crossplane
+  feature_goldenTests: true
+  testMatrix:
+    key: provider
+    entries:
+      - defaults
+      - openshift4
```

Comment:
You probably want to add the files `tests/defaults.yml` and `tests/openshift4.yml` with appropriate default values (they can be empty if the component can be compiled without any additional config), as otherwise the GitHub actions created by modulesync will fail. You can either add those in this PR or in the PR which modulesync will generate.